examples/llm/src/models/mosaic_gpt/mosaic_gpt.py

```diff
@@ -327,7 +328,8 @@ def forward(
             logits *= self.logit_scale
 
         return CausalLMOutputWithPast(logits=logits,
```

Comment:
yeah, for that test I mock the forward function so that i can count the number of times it is called and check that the second call gets a non None past kv input. since im mocking the forward call, i have to provide a return value myself, since it wont actually run the code in forward. lmk if that doesn't make sense.